Main.java

```diff
@@ -25,39 +28,56 @@ public static void main(String[] args) {
 
 		// 3:「3年後の今日が何曜日なのか」を計算して、コンソール画面に表示してみよう。
 		LocalDateTime now = LocalDateTime.now();
-		LocalDateTime localDateTime = now.plusYears(3);
-		DayOfWeek week = localDateTime.getDayOfWeek();
+		LocalDateTime threeYearsLater = now.plusYears(3);
+		DayOfWeek week = threeYearsLater.getDayOfWeek();
 		System.out.println(week);
 
 		System.out.println("------------");
 
 		// 4:「引数が0なら偽。引数が1なら真」とコンソール画面に表示してみよう。
-		int bit = new java.util.Random().nextInt(2);
-		System.out.println(judgment(bit));
+		int bit = new Random().nextInt(2);
+		String bitAnswer = judgement(bit);
+		String bitException = "Error: もう一度繰り返します。";
+		int count = 1;
+		while (bitException.equals(bitAnswer)) {
+			count++;
+			if (count == 3) {
+				System.out.println("Error: 正しく処理できませんでした。強制終了します。");
+				System.exit(0);
```

Comment:
エラー終了なら1で終了が良いですね。
ちなみにSystem#exitは開発現場ではほとんど使いません。
(私は使ったことがない)

ただしプログラムが異常終了したときはコードが1で終了するというのは知っておいて良いです！
linux 終了コード　などで調べてみてください！
```suggestion
				System.exit(1);
```